.github/workflows/frontend-lint-burndown.yml

```diff
@@ -0,0 +1,38 @@
+name: frontend-lint-burndown
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * 1'
+  pull_request:
```

Comment:
we can't really afford to add more workflows / jobs as we're already right at our parallelism limit where more jobs will result in queueing behind itself.  if you can add these commands to an existing job that's a better approach (such as the existing linting job)

Reply:
i put the `pull_request` trigger there just to kick off the build and test it out. I def don't want this to be part of each PR.

Reply:
also updated the description to capture more of the thought process.